modules/slacker-server/src/main/dist/config.yaml

```diff
@@ -1,5 +1,5 @@
 collectors:
```

Comment:
All of the changes here cannot be in the base server pom (the hipchat plugin is in a separate codebase)